services/user.py

```diff
@@ -0,0 +1,55 @@
+from django.contrib.auth import get_user_model
+
+from db.models import User
+
+
+def create_user(
+    username: str,
+    password: str,
+    email: str = None,
+    first_name: str = None,
+    last_name: str = None
+) -> User:
+    new_user = get_user_model().objects.create_user(
+        username=username,
+        password=password
+    )
+
+    if email:
+        new_user.email = email
+    if first_name:
+        new_user.first_name = first_name
+    if last_name:
+        new_user.last_name = last_name
+
+    new_user.save()
+    return new_user
+
+
+def get_user(user_id: int) -> User:
+    return get_user_model().objects.get(pk=user_id)
```

Comment:
The `get` method can raise a `DoesNotExist` exception if the user with the given ID does not exist. Consider handling this exception to provide a more user-friendly error message or behavior.